atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/pool/CassandraClientPoolMetrics.java

```diff
@@ -61,10 +67,14 @@ public void recordConnectionExceptionOnHost(CassandraClientPoolingContainer host
         updateMetricOnAggregateAndHost(hostPool, RequestMetrics::markRequestConnectionException);
     }
 
+    public void recordPoolExhaustion(CassandraClientPoolingContainer unused) {
```

Comment:
Do we need to pass the CassandraClientPoolingContainer if it's unused?

Reply:
I did this more for consistency with the existing methods. Though yeah it might be worth not passing it to not give people the impression that it would be reported